packages/flutter/lib/src/widgets/focus.dart

```diff
@@ -79,8 +96,15 @@ class Focus extends StatefulComponent {
 
   final Widget child;
 
+  /// The key that currently has focus globally in the entire focus tree.
+  ///
+  /// This field is always null when asserts are disabled.
```

Comment:
Since this sounds useful, maybe talk about why you would use it during debugging and how to get the equivalent value in release mode and why we hate the author so much that we don't provide it in release mode. :-)